identity-controller/src/app/services/email.service.ts

```diff
@@ -21,9 +22,26 @@ export class EmailService {
         rejectUnauthorized: false,
       },
       logger: true,
+      
     });
     this.transporter = transport;
     this.adminEmail = 'peter.watkins@gov.bc.ca';
+    } else {
+      const transport = nodemailer.createTransport({
+        host: 'pop3.mailtrap.io',
+        port: 2525,
+        auth: {
+          user: '6969c1013a75cd',
+          pass: '05e096d9af30c8',
```

Comment:
Are these generic credentials provided by mailtrap, or your own? In the latter case we should parametrize them so that they are not hard-coded... Same for the email address a few lines below these.